### ISSUE TITLE
Fairly complex setup

### ISSUE THREAD

**Issue by paul19920801** [gravitl/netmaker#1199]
Hi all

Need some advice here. I have a number of VPS machines, I want to link together. The one is based in New Zealand, and the rest are in Sydney, and in London. My New Zealand VPS is extremely slow, and the only thing beneficial about it is the 100mb connection. Netmaker server works fine, so it was my intention to run a wireguard tunel from NZ to Aus, that way I could use the NZ IP, and use the better speced Aussie VPS I have. 

When I activate egresss mode, I lose connectivity via the public IP, Pings time out and the way to remain connected is to connect via Tailscale, and in some ways that's beneficial, but would also prefer the option of having my devices accessible via the Australian public IP address despite being connected over a VPN type setup.  Then I also want to connect this set to my wider Netmaker Mesh. This server provides streaming channels for personal use. I also see that when everything is connected I have issues with pinging. On some devices I can ping, and others when I ping back, it times out, or halts. 

What's the best way to get this all done?

Thanks

So how

**Comment by mattkasun** [comment]
How have you configured the egress?

**Comment by paul19920801** [comment]
I used the standard egress gateway nodes,  0.0.0.0/5,8.0.0.0/7,11.0.0.0/8,12.0.0.0/6,16.0.0.0/4,32.0.0.0/3,64.0.0.0/2,128.0.0.0/3,160.0.0.0/5,168.0.0.0/6,172.0.0.0/12,172.32.0.0/11,172.64.0.0/10,172.128.0.0/9,173.0.0.0/8,174.0.0.0/7,176.0.0.0/4,192.0.0.0/9,192.128.0.0/11,192.160.0.0/13,192.169.0.0/16,192.170.0.0/15,192.172.0.0/14,192.176.0.0/12,192.192.0.0/10,193.0.0.0/8,194.0.0.0/7,196.0.0.0/6,200.0.0.0/5,208.0.0.0/4

If I need to submit any more info, please let me know.

Thanks for the response.

**Comment by mattkasun** [comment]
Not sure I have a good picture of your network and how you want to use it.  Would you be open to a voice chat to discuss in detail?

**Comment by paul19920801** [comment]
> Not sure I have a good picture of your network and how you want to use it. Would you be open to a voice chat to discuss in detail?

certainly, or via voice note. What application do you suggest for Voicechat? We mainly use WhatsApp in South Africa.

**Comment by mattkasun** [comment]
we have a discord support server
https://discord.gg/VwZCf8rD

**Comment by paul19920801** [comment]
Just getting discord set up, will call you quickly as want this sorted but it's also quite late here, so will give you a basic overview. Hopefully you would then be able use that to advise me in what direction I need to go in.

**Comment by paul19920801** [comment]
Okay, is set up, and am in the group, what next?

**Comment by paul19920801** [comment]
Decided to create a diagram now quickly. Hope this makes sense. 
Some of it I have attempted to set up, whileas others I must still do. However this should give you an better (or hopefully) overall picture of what I am trying to achieve. 

[https://cloud.smartdraw.com/share.aspx/?pubDocShare=4162811332E26B4050667DD4FD24E439F60](url)

**Comment by paul19920801** [comment]
![Network-Diagram-Manual](https://user-images.githubusercontent.com/90729302/173162487-d51eca1b-3088-424c-a72a-2b41eb231f87.jpg)


Here's my diagram. Decided to do it now quickly. Hope this makes more sense.

**Comment by paul19920801** [comment]
I have a script from someone else who had issues with Wireguard, and oracle cloud.

Apparently this corrected routing issues, as can confirm I am unable to ping devices which are on oracle cloud, despite everything being setup correctly. 

I have set everything up by running the autoinstall script, and have disabled the firewalls all together. 

https://pastebin.com/raw/DWRcUjX2
https://pastebin.com/raw/pkf5Vv8Z

https://www.reddit.com/r/WireGuard/comments/oxmcvx/cant_seem_to_get_wireguard_working_on_oracle/


I would try to add these scripts to the Netmaker/Wireguard config myself if I knew where everything was.

**Comment by paul19920801** [comment]
Closed due to too complicated, and rather going to deal with this on a step by step basis
